src/m365/teams/commands/channel/channel-get.ts

```diff
@@ -156,12 +173,24 @@ class TeamsChannelGetCommand extends GraphCommand {
       return `${args.options.teamId} is not a valid GUID`;
     }
 
-    if (args.options.channelId && args.options.channelName) {
-      return 'Specify either channelId or channelName, but not both.';
+    if (args.options.channelId && args.options.channelName && args.options.primary) {
+      return 'Specify channelId or channelName or primary, but not all three.';
```

Comment:
For readability, let's replace all `or` except the last one with commas in all occurrences.

Reply:
Thanks for all the suggestions, updated the validation message.